### PR TITLE
Created profile page using NativeBase.

### PR DESCRIPTION
**Pull request by DougieWougie**
- [Description](#description)
  - [Type of change](#type-of-change)
- [Checklist](#checklist)

# Description
**Draft PR**
- Investigating NativeBase as a design system - added a fourth tab to the NavBar and a Profile Container. Implements most of the functionality of the Example Container but using NativeBase Components.
- Note that dark mode doesn't change the pages theming properly (still working on theming systems)
- Note that changing the theme in Example Container will not update the Radio Group in Profile Container because Example Container doesn't set a value in state.
- Adds native-base 3.4.21.
- Adds react-native-svg 12.1.1.
- Added a sample contianer to replace EventContainer and ProjectContainer.
- Sample components added to container.

## Type of change

Please delete options that are not relevant
- [] Bug fix (non-breaking change which fixes an issue)
- [X] New Feature (non-breaking change which adds functionality)
- [] Breaking change(fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation (if required)
- [x] My changes generate no new warnings
- [] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by rc-1988 — APPROVED** [review]
Nice work, @Dougie - should get us started with a strong foundation for the design implementation :-)

For when we've forgotten what happened with this: merging to get us started with NativeBase. Since Dougie's away I've pushed a couple of commits, with his blessing, that implement things he'd suggested doing in his comments.

File: app/src/NativeBase/Containers/VerticalStackContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
+/**
```
Comment (joefhall) [inline app/src/NativeBase/Containers/VerticalStackContainer.tsx:1]: Is this an example container to show how to do one?  If so, wonder if we should remove it -- or if it's helpful to leave it in for now, do we need another ticket to make sure we take it out later?

Comment (rc-1988) [reply app/src/NativeBase/Containers/VerticalStackContainer.tsx:1]: It's a good demo example, I think worth leaving in for demo purposes for now until we've got more of the codebase moved over. Good shout to raise a ticket to remove it later though; I'll raise that now.

Comment (joefhall) [reply app/src/NativeBase/Containers/VerticalStackContainer.tsx:1]: Nice one, thanks 👍

**Review by PammyLo — APPROVED** [review]
Great work @DougieWougie and @rc-1988, thank you! 
Works well in iOs and Android.

**Pull request merged**